app/src/ui/clone-repository/clone-repository.tsx

```diff
@@ -218,7 +218,7 @@ export class CloneRepository extends React.Component<
           selectedIndex={this.props.selectedTab}
         >
           <span>GitHub.com</span>
-          <span>Enterprise</span>
+          <span>Enterprise Server</span>
```

Comment:
I'm not 100% if this is purely cosmetic 💅 or if changing this tab label has a broader impact.

Reply:
👍 we use a `CloneRepositoryTab` enum to represent the selected tab, so a label change should be safe.